src/OrgV1.sol

```diff
@@ -6,12 +6,15 @@ interface IERC20 {
 }
 
 interface ENS {
-    function owner(bytes32 node) external view returns (address);
-    function resolver(bytes32 node) external view returns (address);
+    function owner(bytes32) external view returns (address);
```

Comment:
https://docs.ens.domains/contract-api-reference/ens

Add method parameter names?

```solidity
function owner(bytes32 node) external view returns (address);
function resolver(bytes32 node) external view returns (address);
function setOwner(bytes32 node, address owner) external;
function setSubnodeOwner(bytes32 node, bytes32 label, address owner) external;
```


do we need to additional ENS method? e.g. add `ttl`?

```solidity
function ttl(bytes32 node) external view returns (uint64);
```